src/api/java/io/github/cvc5/Solver.java

```diff
@@ -2180,6 +2185,42 @@ public Term declarePool(String symbol, Sort sort, Term[] initValue)
   private native long declarePool(
       long pointer, String symbol, long sortPointer, long[] termPointers);
 
+  /**
+   * Declare an oracle function with reference to an implementation.
+   *
+   * Oracle functions have a different semantics with respect to ordinary
+   * declared functions. In particular, for an input to be satisfiable,
+   * its oracle functions are implicitly universally quantified.
+   *
+   * This method is used in part for implementing this command:
+   *
+   * {@code
+   * (declare-oracle-fun <sym> (<sort>*) <sort> <sym>)
+   * }
+   *
+   *
+   * In particular, the above command is implemented by constructing a
+   * function over terms that wraps a call to binary sym via a text interface.
+   *
+   * @api.note This method is experimental and may change in future versions.
+   *
+   * @param symbol The name of the pool
```

Comment:
```suggestion
   * @param symbol The name of the oracle
```

Reply:
@ajreynol  This is copied from the cpp api documentation.

Reply:
I remember adding this suggestion to Andy's PR too. I think it says `oracle` now.